diff --git a/torch/_inductor/graph.py b/torch/_inductor/graph.py
index d10dc7a..11b6da1 100644
--- a/torch/_inductor/graph.py
+++ b/torch/_inductor/graph.py
@@ -1335,10 +1335,10 @@ class GraphLowering(torch.fx.Interpreter):
             return self.add_tensor_constant(value, target)
 
         with no_dispatch():
-            if value.shape == ():
-                return Constant(
-                    value=value.item(), dtype=value.dtype, device=value.device
-                )
+            # if value.shape == ():
+            #     return Constant(
+            #         value=value.item(), dtype=value.dtype, device=value.device
+            #     )
             if self.can_inline_constant(value):
                 log.debug("Inlining constant: %s ", str(target))
                 # tensor lowering has constant inlining logic
@@ -2134,7 +2134,7 @@ class GraphLowering(torch.fx.Interpreter):
         """
         For GPU, Triton kernels are autotuned and stored as cubin files
         """
-        if any(device in self.device_types for device in ["cuda", "xpu"]):
+        if any(device in self.device_types for device in ["cuda", "xpu", "musa"]):
 
             def extract_real_inputs() -> list[Union[int, float, torch.Tensor]]:
                 def materialize(
