diff --git a/torch/_inductor/runtime/triton_compat.py b/torch/_inductor/runtime/triton_compat.py
index 645e0f4..6f97f35 100644
--- a/torch/_inductor/runtime/triton_compat.py
+++ b/torch/_inductor/runtime/triton_compat.py
@@ -145,6 +145,8 @@ def cc_warp_size(cc: Union[str, int]) -> int:
             return 32
         else:
             return 64
+    elif hasattr(torch, "musa") and int(str(cc)) < 31:
+        return 128
     else:
         return 32
 
