diff --git a/torch/_inductor/cpp_builder.py b/torch/_inductor/cpp_builder.py
index e2cb445..5a82dfd 100644
--- a/torch/_inductor/cpp_builder.py
+++ b/torch/_inductor/cpp_builder.py
@@ -1603,6 +1603,13 @@ def get_cpp_torch_device_options(
             if not find_library("ze_loader"):
                 raise OSError(xpu_error_string)
 
+    if device_type == "musa":
+        definitions.append(" USE_MUSA")
+        from torch_musa.utils import musa_extension
+        include_dirs = musa_extension.include_paths(True)
+        libraries_dirs = musa_extension.library_paths(True)
+        libraries += ["musa", "musa_python"]
+
     if device_type == "mps":
         definitions.append(" USE_MPS")
 
